consensus/src/rand/rand_gen/rand_store.rs

```diff
@@ -133,13 +144,23 @@ impl<S: TShare> RandItem<S> {
         }
     }
 
-    fn try_aggregate(&mut self, rand_config: &RandConfig, decision_tx: Sender<Randomness>) {
+    fn try_aggregate(
+        &mut self,
+        rand_config: &RandConfig,
+        decision_tx: Sender<Randomness>,
+        is_fast_path: bool,
```

Comment:
nit: it's better to have this as part of the share aggregator constructor instead of passing in on try_aggregator.